test/unit/mdc-textfield/foundation.test.js

```diff
@@ -173,6 +174,28 @@ test('#init does not add mdc-textfield__label--float-above class if the input do
   td.verify(mockAdapter.addClassToLabel(cssClasses.LABEL_FLOAT_ABOVE), {times: 0});
 });
 
+test('#setHelptextValue has no effect when no help text element is present', () => {
+  const {foundation} = setupTest();
+  foundation.setHelptextValue('foo');
+});
+
+test('#setHelptextValue sets the value of the help text element', () => {
+  const {foundation, mockAdapter} = setupTest();
+  foundation.setHelptextValue('foo');
+  td.verify(mockAdapter.setHelptextValue('foo'));
+});
+
+test('#getHelptextValue has no effect when no help text element is present', () => {
+  const {foundation} = setupTest();
+  foundation.getHelptextValue();
```

Comment:
Same comment as above

Reply:
Same as above, moved it to the `mdc-text-field.test.js` file

https://github.com/patrickrodee/material-components-web/blob/37c078df5b21d597076636337e62a1a67494d59c/test/unit/mdc-textfield/mdc-text-field.test.js#L162-L165